languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -35467,6 +35467,37 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TOURS_CRITIQUES" name="Tours critiqués divers (barbarismes, impropriétés, solécismes, etc.)">
+      <rulegroup id="ARGOT_BEAUF_ET_BOF" name="argot beauf et bof" default="temp_off">
+          <rule>
+              <pattern case_sensitive="yes">
+                  <marker>
+                      <token regexp="yes">bof</token>
```

Comment:
Couldn't the interjection "bof" also occur after the beginning of a sentence? "Je n'en sais rien, bof..."

Reply:
Any feedback on this?

Reply:
I've looked at the rule editor results for this. We need to address this before we can activate it.
We need a `<token negate_pos="yes" postag="SENT_START">`. Also, we need an AP for "bof bof". The sentence "Bof bof." (here, "bof" is not an adjective, but an interjection, as explained) should neither become "Moyen moyen." nor "Bof moyen.", but stay unchanged. We should keep our hands of "bof" as an interjection. "Moyen" can be a good replacement for that **sometimes**, but it is pure random.